modules/typelevelShell.nix

```diff
@@ -53,13 +54,10 @@ in
             ${bold}[versions]${reset}
 
               Java - ${cfg.jdk.package.version}
-          '' + optionalString cfg.nodejs.enable "  Node - ${pkgs.nodejs-16_x.version}\n";
+          '' + optionalString cfg.nodejs.enable "  Node - ${cfg.nodejs.package.version}\n";
```

Comment:
I don't love this coupling, but I think we'd need another option for versions, and then refer to that, to get modules to merge. :shrug: